### PR TITLE
2 simplistic fastapi route to create users

### LINKED ISSUE #2: Simplistic FastAPI route to create users

### PR DESCRIPTION
**Pull request by Yousifff**
Adding main route ("/") to get data from database and display it.
Saving user from admin frontend page to backend route "/admin"

### CODE REVIEW

File: backend/__pycache__/app.cpython-312.pyc
Comment (Zaubeerer) [inline backend/__pycache__/app.cpython-312.pyc:1]: make sure not to add these cache files to the git

File: backend/app.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+@asynccontextmanager
+async def lifespan(app: FastAPI):
+    create_db_and_tables()
+    yield
+
+
+app = FastAPI(lifespan=lifespan)
+
+
+@app.post("/admin")
```
Comment (Zaubeerer) [inline backend/app.py]: This is fine for now, in a next step you could create a router for `users` and just add the routes to that router :)

File: backend/database.db
Comment (Zaubeerer) [inline backend/database.db:1]: should not be committed to the repo

File: frontend/.streamlit/confit.toml
Comment (Zaubeerer) [inline frontend/.streamlit/confit.toml:1]: I guess this has a typo and should rather be called `config.toml` ;-)

File: frontend/pages/admin.py [hunk 2]
```diff
@@ -9,10 +9,16 @@ def create_form():
         dec_31 = datetime.date(today.year,12,31)
         name = st.text_input("Name",placeholder='Name')
         schdule_date = st.date_input("date",(jan_1,dec_31))
+        url = "http://127.0.0.1:8000/admin"
         submit = st.form_submit_button("Save")
         if submit:
-            st.session_state['database'][name] = schdule_date
+            st.write(name.strip())
+            data = {'name': name}
+            requests.post(url,json=data)
```
Comment (Zaubeerer) [inline frontend/pages/admin.py:18]: nice! 👍

File: frontend/pixi.toml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+[project]
+name = "frontend"
+version = "0.1.0"
+description = "Add a short description here"
+authors = ["Yousifff <yousifsayer@gmail.com>"]
+channels = ["conda-forge"]
+platforms = ["linux-64"]
+
+[tasks]
+admin = "pixi run streamlit run pages/admin.py"
```
Comment (Zaubeerer) [inline frontend/pixi.toml:10]: You should just create a task `app = "pixi run streamlit run main.py"`

Streamlit will then identify the pages and run the app appropriately.

**Review by Zaubeerer — CHANGES_REQUESTED** [review]
Good PR, nice that you added pixi to frontend 👍 

I have just a couple of small requests, then we can merge and you can continue with further functionalities in new feature branches.

File: backend/app.py (hunk 1 above)
Comment (Yousifff) [reply backend/app.py]: Do you mean user router is "/user" ?

File: backend/__pycache__/app.cpython-312.pyc
Comment (Yousifff) [reply backend/__pycache__/app.cpython-312.pyc:1]: Fixed

File: backend/database.db
Comment (Yousifff) [reply backend/database.db:1]: Fixed

File: frontend/pixi.toml (hunk 3 above)
Comment (Yousifff) [reply frontend/pixi.toml:10]: Yes, it's raising error actually this why I added admin and index in pixi file.
I'm trying to solve it.

File: backend/app.py (hunk 1 above)
Comment (Yousifff) [reply backend/app.py]: /admin route has been changed to /user

**Review by Zaubeerer — APPROVED** [review]
There are still caching files in the PR, but you can merge it and we can fix it together next session

File: frontend/pixi.toml (hunk 3 above)
Comment (Zaubeerer) [reply frontend/pixi.toml:10]: OK, I am wondering why you had this problem 🤔

Anyway, as mentioned in the call, I suggest to switch to the frontend setup I proposed in the previous app, so you have login etc.

**Comment by Yousifff** [comment]
> There are still caching files in the PR, but you can merge it and we can fix it together next session

Fixed

**Comment by Zaubeerer** [comment]
Feel free to merge :)

**Pull request merged**